dart/send-http-request/README.md

```diff
@@ -0,0 +1,50 @@
+# 📧 Sending HTTP Request
+
+A sample Dart Cloud Function for sending HTTP request.
+
+_Example input:_
+
+```json
+{
+    "payload": "{\"url\": \" <YOUR-URL> \", \"method\": \"GET\"}",
+    "headers": {
+        "x-client-version": "1.0.0"
+    }
```

Comment:
headers is actually supposed to be part of the payload like URL, method, and body.